dialogue-java-client/src/main/java/com/palantir/dialogue/HttpChannel.java

```diff
@@ -118,6 +119,11 @@ public int code() {
                 return response.statusCode();
             }
 
+            @Override
+            public Map<String, List<String>> headers() {
+                return response.headers().map();
```

Comment:
idk ... is it worth a defensive copy here?

Reply:
Can verify that this is an unmodifiable map: https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/master/src/java.net.http/share/classes/java/net/http/HttpHeaders.java#L117